ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -123,6 +128,26 @@ def get_anta_results(
     )
 
 
+def load_custom_catalogs(catalog_files: list[Path]) -> dict:
+    """Load the custom ANTA catalogs from the provided files and merge them together.
+
+    Args:
+    ----
+      catalog_files (list[Path]): List of Path objects pointing to the custom ANTA catalog files.
+
+    Returns:
+    -------
+      dict: The merged custom ANTA catalog.
+    """
+    catalog_list = []
+    for file in catalog_files:
+        with file.open("r", encoding="UTF-8") as fd:
+            catalog = load(fd, Loader=CSafeLoader)
+            catalog_list.append(catalog)
```

Comment:
exception handling - what if the file opening is causing issue. e.g. permission denied or malformated yaml?

Reply:
We catch all exceptions in the runner. The task will fail for the device that is trying to open the catalog only:
![image](https://github.com/aristanetworks/avd/assets/63206086/18176235-ae09-4d0b-ac2c-01a6ced163af)

Reply:
Fixed in 1461c6d05b7bc8f4c5cc65f88af4b0215336d516